front/ticket.form.php

```diff
@@ -68,7 +68,11 @@
    Html::back();
 
 } else if (isset($_POST['update'])) {
-   $track->check($_POST['id'], UPDATE);
+   if ($track->canAssign()) {
+      $track->check($_POST['id'], READ);
+   } else {
+      $track->check($_POST['id'], UPDATE);
+   }
```

Comment:
Maybe I am wrong, but this would permit anyone that has only ticket read rights to update any ticket with a custom script that simulate a form post.